examples/source/2.guides/Using_the_Google_AMP_Cache/api.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+const utils = require('@lib/utils');
+const {setMaxAge} = require('@lib/utils/cacheHelpers');
+const nunjucks = require('nunjucks');
+
+// eslint-disable-next-line new-cap
+const examples = express.Router();
+
+const filePath = utils.project.absolute('/examples/static/samples/files/get-example.html');
+
+examples.get('/g', (request, response) => {
```

Comment:
can you rename this to `s/g/timestamp`

Reply:
Did you mean ```/s/g/timestamp``` or just ```/g/timestamp```?

Reply:
Went with ```/g/timestamp```. Please let me know, if it is supposed to be ```s/g/timestamp```.